### PR TITLE
docs(optimization) Add optimization section to configuration docs

### PR DESCRIPTION
**Pull request by EugeneHlushko**
Start adding `optimize` section to the configuration

Tackles first four from #1968 

Suggest we merge this one with those few checkboxes tackled, so more people could contribute to the new page and finish #1968 quicker

### CODE REVIEW

File: src/content/configuration/optimization.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Optimization
+sort: 8
+contributors:
+  - EugeneHlushko
+---
+
+Optimization configuration allows you to control the webpack output optimization level
```
Comment (montogeek) [inline src/content/configuration/optimization.md]: Add final dot.

File: src/content/configuration/optimization.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+## `optimization.minimize`
+
+`boolean`
+
+Tell webpack to minimize the javascript output using the [UglifyjsWebpackPlugin](/plugins/uglifyjs-webpack-plugin/)
```
[the remark this answers is not in the document]
Comment (jeremenichelli) [reply src/content/configuration/optimization.md]: Isn't the plugin deprecated? I would also rephrase this as:

_Tell webpack to minimize the output bundles using [UglifyJS](https://github.com/mishoo/UglifyJS2)._

[the remark this answers is not in the document]
Comment (EugeneHlushko) [reply src/content/configuration/optimization.md]: Its not deprecated, you still provide it as minimizer? @ooflorent

[the remark this answers is not in the document]
Comment (EugeneHlushko) [reply src/content/configuration/optimization.md]: https://github.com/webpack/webpack/blob/master/lib/WebpackOptionsDefaulter.js#L277

**Comment by EugeneHlushko** [comment]
I've updated syntax things, the plugin is untouched i'm pretty sure you still can use it and its being used! @montogeek @jeremenichelli

**Comment by EugeneHlushko** [comment]
@montogeek @jeremenichelli this been hanging for a while, any comments or good to merge?

**Comment by montogeek** [comment]
Thanks @EugeneHlushko ! :D

**Comment by EugeneHlushko** [comment]
🎉

**Comment by jeremenichelli** [comment]
Sorry I'm late @EugeneHlushko, I have a couple of corrections for this PR. Do you want me to put them here and you can create a new PR or do you want me to do it and tag you as a reviewer?
